.github/workflows/needs-decision.yml

```diff
@@ -0,0 +1,46 @@
+name: Needs Decision
+# Post a comment on Issues to explain what the "Needs Decision" label means.
+
+permissions:
+  contents: read
+  issues: write
+
+on:
+  issues:
+    types:
+      - labeled
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GH_REPO: ${{ github.repository }}
+  ISSUE_NUMBER: ${{ github.event.issue.number }}
+
+jobs:
+
+  post_comment:
+    name: Add 'Needs Decision' comment
+    if: github.event.label.name == 'Needs Decision'
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: add 'Needs decision' comment
+        run: |
+          gh api \
+          --method POST \
+          -H "Accept: application/vnd.github+json" \
+          -H "X-GitHub-Api-Version: 2022-11-28" \
+          /repos/$GH_REPO/issues/$ISSUE_NUMBER/comments \
+          -f "body=$BODY"
+        env:
+          BODY: |
+            Thanks for the work you've done so far. The goal of this comment
+            is to set expectations.
+
+            Deciding on new features or substantial changes is a lengthy
+            process. It frequently happens that no maintainer is available
```

Comment:
Do you think it is worth adding background information on how a "new feature is like giving someone a puppy" - often people think a new feature only requires the work of adding a new feature (why not 'just' add this?) but cannot see the longer term maintenance required.

Reply:
I think that is too much level of detail. The longer the text, the lower the chances of people reading it. The way I think about these things is along the lines of checklists (as used in aviation). They aim to remind pilots of the things that need to be done, but if you don't know how to do the thing they are useless.

<img width="668" height="800" alt="image" src="https://github.com/user-attachments/assets/3385126b-61eb-4b48-bb5c-17885fddc090" />

For me this is a reminder to everyone, including maintainers, not a moment to teach people. If you want to go fly you probably know what "Cockpit prep" means, and if not you should ask. But I assume that question means you won't fly today and instead go away to a classroom to learn about it.